next-tavla/src/Admin/components/FooterTextInput/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { TextField } from '@entur/form'
+import { useSettingsDispatch } from 'Admin/utils/contexts'
+import classes from './styles.module.css'
+
+function FooterTextInput({ footer = '' }: { footer?: string }) {
+    const dispatch = useSettingsDispatch()
+
+    return (
+        <div className={classes.footerInput}>
+            <TextField
+                onChange={(e) =>
+                    dispatch({
+                        type: 'addFooter',
+                        footer: e.target.value,
+                    })
+                }
+                size="medium"
+                label="Footer"
+                value={footer}
+                clearable={true}
+                onClear={() =>
+                    dispatch({
+                        type: 'addFooter',
+                        footer: '',
+                    })
+                }
```

Comment:
Forsåvidt greit å bruke addFooter til å fjerne, men det er litt misvisende. Kunne vurdert en ekstra action for å fjerne/cleare